pkg/runtime/kubernetes/init.go

```diff
@@ -237,9 +241,22 @@ func (k *Runtime) initKube(hosts []net.IP) error {
 
 func (k *Runtime) sendClusterCert(hosts []net.IP) error {
 	f := func(host net.IP) error {
-		err := k.infra.Copy(host, k.getPKIPath(), clustercert.KubeDefaultCertPath)
-		if err != nil {
-			return fmt.Errorf("failed to copy cluster cert : %v", err)
+		if err := k.infra.Copy(host, k.getPKIPath(), clustercert.KubeDefaultCertPath); err != nil {
+			return fmt.Errorf("failed to copy cluster cert: %v", err)
+		}
+		if err := k.infra.Copy(host, k.getPKIPath(), k.getPKIPath()); err != nil {
```

Comment:
What does this mean?

Reply:
transform pki to all master's rootfs, so when master0 down, other master can scale up cluster.